model/place.go

```diff
@@ -23,13 +23,15 @@ type Uint32Slice []uint32
 
 // Place holds information about a place
 type Place struct {
-	ID               uint32      `json:"id" dynamodbav:"pk"`
+	ID               uint32      `json:"id" dynamodbav:"pk,string"`
+	Type             string      `json:"-" dynamodbav:"sk"`
```

Comment:
Just for my understanding, these fields: Place.Type + AltSort, PlaceWord.Pk + Type, and PlaceSettings.Pk + Sk are only used if the backend is dynamodb?

Reply:
Yes, the ones with `dynamodbav ` and `json:"-"` annotations are DynamoDB-specific.